### PR TITLE
Support new mkdocstrings_handlers namespace

### PR DESCRIPTION
**Pull request by pawamoy**

### CODE REVIEW

**Comment by pawamoy** [comment]
Let's not emit deprecation warnings for our own legacy handler. Users don't care about them.

**Comment by pawamoy** [comment]
@oprypin do you want us not to emit these deprecation warnings for the crystal handler as well?

**Comment by oprypin** [comment]
@pawamoy Well yes but that will just be done by making the handler use the new approach

**Comment by pawamoy** [comment]
Indeed, it's just in case you don't plan to upgrade your handler immediately and don't want to bother your users with a deprecation warning they have no control on. Let me know.

File: docs/handlers/overview.md [hunk 1]
```diff
@@ -42,7 +40,7 @@ See the documentation for
 [`BaseRenderer`][mkdocstrings.handlers.base.BaseRenderer].
 
 Check out how the
-[Python handler](https://github.com/pawamoy/mkdocstrings/blob/master/src/mkdocstrings/handlers/python.py)
+[Python handler](https://github.com/mkdocstrings/python/blob/master/src/mkdocstrings_handlers/python)
```
Comment (oprypin) [inline docs/handlers/overview.md:43]: https://github.com/mkdocstrings/mkdocstrings/pull/356 will remove this destination though

Comment (pawamoy) [reply docs/handlers/overview.md:43]: Hmmm no I think it's right: this link points to the extracted Python handler, i.e. its own repository in the org, and not mkdocstrings' repo.

**Pull request merged**